src/views/PerOverviewForm/index.tsx

```diff
@@ -153,12 +149,13 @@ export function Component() {
         },
     });
 
-    useRequest<LatestPerOverviewResponse>({
+    useRequest({
         url: '/api/v2/latest-per-overview/',
         skip: isNotDefined(value?.country) || isDefined(value?.is_draft),
         query: {
-            country_id: value?.country,
-        },
+            country_id: Number(value?.country),
+            // FIXME: typing for query not available
+        } as never,
```

Comment:
Why cast as never?

Reply:
Becase the query typing doesn't exsit for this end-point

Reply:
Why not cast as `any`?

Reply:
We'll need one additional line of ignore comment